### ISSUE TITLE
Azure Function app stops running

### ISSUE THREAD

**Issue by komainu85** [Azure/azure-webjobs-sdk-script#2158]
I'm actually experiencing an issue currently with one of my timer functions. It should run ever hour with this hourly schedule "0 0 * * * *", it works for a few hours then stops. A restart starts it working again, after a few hours it stops again. Sometimes it can run twice in an hour, with a few minutes gap.

#### Investigative information

Please provide the following:

- Function App version (1.0 or 2.0-beta): 1.0.11388.0
- Function App name: MikeRetweetBot
- Function name(s) (as appropriate): Retweet
- Region: UK South

#### Expected behavior

The function should run every hour.

#### Actual behavior

Runs every hour for a few hours, then the function stops running. At times the function can fire twice within an hour within a few minutes of the previous run, ignoring the schedule. Ive also seen the function run up to 20 minute late of the schedule.

**Comment by GrzesiekGdn** [comment]
We have the same problem with our functions.

We have four azure functions with only timer triggers, they are created on dedicated consumer plan (we also have another app service plan). They are schedulled by different frequencies: from every 15 munutes to once a month.

Two functions are written using cs scripts (csx), and another two using f# scripts (fsx). All are very simple and manually deployed to Azure (using function editor).

#### Investigative information
- Function App version (1.0 or 2.0-beta): 1.0.11388.0 (~1)
- Function App name: wcn-ga
- Function name(s) (as appropriate): UpdateGaStats
- Region: NorthEurope

#### Expected behavior
The function should run every 15 minutes.

#### Actual behavior
Functions are run only after opening function in azure portal.

**Comment by christopheranderson** [comment]
Is this still reproing? Sorry we missed this over the thanksgiving/holiday season. We don't have the logs to investigate what might have gone wrong during this time period, but it looks like a few other people are reporting similar issues. For service impacting issues, I generally recommend using the Support ticket system in the Azure Portal since that has an SLA (and github is just best effort, which can be 0 during holidays, unfortunately....). If it's still reproing, please open a fresh issue so it's on the top of our stack.

**Comment by GrzesiekGdn** [comment]
Yes it is still not working properly. For example:
![image](https://user-images.githubusercontent.com/11981612/35086994-dbd0dc94-fc2e-11e7-8db6-b0460a3acabe.png)
As you see in last month the function was runned only few times instead few hundreds (cron expression: 0 */15 * * * *).
I have created new function app wcn-lambdas with the same resource group as main application on app service plan and the new instance is working ok.
But is it normal that lambda function triggered by timer with consumption plan isn't working in reliable way?
Regards,
Grzegorz

**Comment by komainu85** [comment]
Still occurring for me, so opened a new issue #2317

**Comment by arefinsami** [comment]
Turn on the "AlwaysOn" setting in the configuration.
